### PR TITLE
Adding new source func

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

File: tests/test_live.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def test_instruments():
         host=get_platform_dispatcher(),
         instrument="mock",
     )
-    assert disp.get_instruments_list() == ['magic', 'isgri', 'jemx', 'polar', 'antares', 'spi_acs']
+    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'antares', 'spi_acs']
```
Comment (burnout87) [inline tests/test_live.py:90]: This test, now passes

**Review by volodymyrss — APPROVED** [review]

File: tests/test_live.py (hunk 1 above)
Comment (burnout87) [reply tests/test_live.py:90]: any ideas @volodymyrss  ?

Comment (volodymyrss) [reply tests/test_live.py:90]: what is the problem precisely?

Comment (burnout87) [reply tests/test_live.py:90]: the list of instruments returned does not includ `magic`

Comment (volodymyrss) [reply tests/test_live.py:90]: it should not. we do not provide magic.

**Pull request merged**